planner/core/planbuilder.go

```diff
@@ -1028,6 +1039,10 @@ func getPossibleAccessPaths(ctx sessionctx.Context, tableHints *tableHintInfo, i
 				ctx.GetSessionVars().StmtCtx.AppendWarning(err)
 				continue
 			}
+			if isGlobalIndex(idxName, tblInfo) {
```

Comment:
This change means do not use the global index for all the cases...
Can we just avoid using it for some special cases?